lib/main.dart

```diff
@@ -62,16 +64,20 @@ Future<void> main() async {
   Utils.packageInfo = await PackageInfo.fromPlatform();
 
   await Firebase.initializeApp();
+  final remoteConfigService = await RemoteConfigService.getInstance();
+  await remoteConfigService.initialise();
```

Comment:
Prefer doing async operations in `setUpAndChooseStartScreen` if possible, when we're actually showing the splash loading screen. Everything in main here will slow down the app launch.